lib/cubits/person_account/person_account_cubit.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:developer';
+
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+
+import '../../models/person_account.dart';
+import '../../services/person_service.dart';
+
+part 'person_account_state.dart';
+
+class PersonAccountCubit extends Cubit<PersonAccountState> {
+  final PersonService personService;
+
+  PersonAccountCubit({
+    required this.personService,
+  }) : super(PersonAccountInitial());
+
+  Future<void> getPersonAccounts() async {
+    try {
+      String personId = 'mockpersonkontistgmbh';
```

Comment:
What would you say to move into a constant in a separate file, called something like `temporary_harcoded_values` so we know that we have to get back and replace them.

You can even add a [TD] (technical debt) item in JIRA that will remind you to come back to this.
@szaszgyozo